lib/mollie/api/client.rb

```diff
@@ -89,10 +70,9 @@ def perform_http_call(http_method, api_method, id = nil, http_body = {})
             raise Mollie::API::Exception.new("Invalid HTTP Method: #{http_method}")
         end
 
-        request.add_field('Accept-Encoding', 'application/json')
-        request.add_field('Authorization', "Bearer #{@api_key}")
-        request.add_field('User-Agent', @version_strings.join(" "))
-        request.add_field('X-Mollie-Client-Info', uname)
+        request['Accept']        = 'application/json'
+        request['Authorization'] = "Bearer #{@api_key}"
+        request['User-Agent']    = "Mollie-Ruby/#{VERSION}"
```

Comment:
I try really hard to hide my version information on my servers, because it makes attacks to a system a lot easier. So I wouldn't be too happy about sending this information to a service I have no control over.